streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsProducer.java

```diff
@@ -212,17 +190,6 @@ public void flush() {
         producer.flush();
     }
 
-    public void close() {
```

Comment:
No one who has a reference to the StreamsProducer has any business calling close, so the method is gone now.